.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: ci
+on:
+  push:
+    branches:
+      - develop
+      - release/**
+      - hotfix/**
+      - feat/dev-245-build-binaries
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
```

Comment:
I don't know how glibc compatibility works on Ubuntu, so I don't know if `ubuntu-latest` might cause any issues, but if the binary works on Ubuntu 16.04, we should be good.

Reply:
I wondered that same thing.  I figured we can try this and test out the client.  If there's issues, then we can submit another pr to use `ubuntu-16.04` instead